docs/_documentation/presenters/android-view-presenter.md

```diff
@@ -29,27 +30,57 @@ The presenter uses a set of `PresentationAttributes` to define how a view will b
 
 ### MvxActivityPresentationAttribute
 
-Used as root of the App and root for other layouts like Fragments.
+Use this attribute if you want to display an Activity in your application. An Activity will be the root of your application and also can act as a host for fragments. Alongside the attribute, your view can customize the presentation by using these attribute properties:
+
+- Extras: Use this `Bundle` to add any extra parameters to the Activity Intent.
+- SharedElements: Consists on a `IDictionary<string, View>` that you can use to add shared view elements to the transition. When using the AppCompat version, the string keys are not relevant.
+
+### MvxFragmentPresentationAttribute
+
+A Fragment is hosted inside an Activity (or a fragment). By using this ViewPresenter, you can decide whether to make all of your screens Activities, or to use an Activity host and many Fragments inside of it. The framework will then help you setting up the navigation and backstack.
```

Comment:
This sentence seems weird to me: "will then help you setting up the navigation and backstack"